packages/shadydom/src/patch-events.js

```diff
@@ -271,6 +272,29 @@ let EventPatches = {
   },
 };
 
+const hasDescriptors = utils.settings.hasDescriptors;
```

Comment:
Is this related to listener options?

Reply:
Safari 9 has broken DOM descriptors and this catches that. Added a comment.